examples/framework/preferences/minimalFootprint/fullPage.html

```diff
@@ -22,13 +22,13 @@
             it makes reference to the individual JavaScript files instead of using
             the built single concatenated JavaScript file. However in a context
             where the built file is available these could all be replaced by a reference to it.
-            e.g. From NPM - "../../../../dist/infusion-all.js"
+            e.g. From NPM - "../../../../dist/infusion-uio.js"
```

Comment:
It's tricky about what name should be used here. [Instructions on npm](https://www.npmjs.com/package/infusion) leads to the creation of "infusion-all.js" or "infusion-custom.js". [Our release page](https://github.com/fluid-project/infusion/releases) uses the name "infusion-uiOptions.js" for UIO build. Feels to me it probably makes more sense to continue using "infuson-all.js" here.

Also, a similar comment is [here](https://github.com/jobara/infusion/blob/FLUID-6322/examples/framework/preferences/minimalFootprint/index.html#L31), which should be updated accordingly if you decided to use another name.

Reply:
It's probably more accurate to be infusion-uio.js as this is a prefs framework example. Although either should technically work. The other examples you mention refer to manually creating the builds through the grunt build process. Where as this is one of the pre-built dists. We'll probably want to change the name on the release page for the next release.